decidim-elections/app/controllers/decidim/elections/admin/steps_controller.rb

```diff
@@ -75,6 +79,17 @@ def elections
         def election
           @election ||= elections.find_by(id: params[:election_id])
         end
+
+        def vote_stats
+          @vote_stats ||= {
+            votes: vote_counts.first,
+            voters: vote_counts.last
+          }
+        end
+
+        def vote_counts
+          @vote_counts ||= election.votes.where(status: "accepted").pluck("COUNT(id)", "COUNT(distinct voter_id)").first
```

Comment:
Could you move this to a method in `election`?

Reply:
Sure! 👍

Reply:
Oh, actually, I added a query object for it. You added the comment to an outdated version. Do you prefer having it as a method in `elections` or keep it as a query? @mrcasals

Reply:
Ah, damn, sorry!!! The query object's fine, leave it as it is then! 😄